.github/workflows/ci.yml

```diff
@@ -18,24 +18,28 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18, 20]
+        node-version: [16, 18, 20]
```

Comment:
Support for 16 got dropped. Not sure if the change was intentional.

Reply:
Oops while installing packages, I think it got omitted because 16 LTS is now out of support. Will add it !

Reply:
Well, actually the other way round. Should kept being removed. ;)

Reply:
I am not sure why it happened .Will do that!